src/cli/cli.cc

```diff
@@ -1363,15 +1418,24 @@ int main (const int argc, const char* argv[]) {
       }
     }
 
-    if (flagRunUserBuildOnly == false && fs::exists(paths.platformSpecificOutputPath)) {
-      auto p = fs::current_path() / fs::path(paths.platformSpecificOutputPath);
+    auto removePath = paths.platformSpecificOutputPath;
+      if (targetPlatform == "android")
+        removePath = settings["output"] + "/android/app/src";
```

Comment:
is this path with `/` slashes okay for windows?

Reply:
Yes, it just works ;)